src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -301,13 +301,21 @@ public void verifyGptIframe(String adUnit, String slotName, String src) {
    * @param src         the source of an ad, for example gpt, remnant or empty
    */
   public String buildGptIframeId(int dfpClientId, String adUnit, String slotName, String... src) {
-    return Joiner.on("/").skipNulls().join(
-        "google_ads_iframe_",
-        String.valueOf(dfpClientId),
-        adUnit,
-        src.length > 0 ? src[0] : null,
-        slotName + "_0"
-    );
+    if (slotName == "BOTTOM_LEADERBOARD"){
+      return Joiner.on("/").skipNulls().join(
+              "google_ads_iframe_",
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "google_ads_iframe_" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)